### PR TITLE
Update telemetry events

### PR DESCRIPTION
**Pull request by sandy081**
- add comments to telemetry events
- remove events not needed

### CODE REVIEW

**Review by lramos15 — CHANGES_REQUESTED** [review]
Thanks for this,

Properties also need to have comments to help explain what they are for example 
`sortBy: { classification: 'SystemMetaData'; purpose: 'FeatureInsight'; comment: 'Sort order, i.e. asc or desc' };`

**Comment by sandy081** [comment]
@lramos15 Added !

File: src/vs/platform/extensionManagement/common/extensionGalleryService.ts [hunk 1]
```diff
@@ -236,20 +236,21 @@ const DefaultQueryState: IQueryState = {
...
-	readonly duration: { classification: 'SystemMetaData'; purpose: 'PerformanceAndHealth'; 'isMeasurement': true };
-	readonly success: { classification: 'SystemMetaData'; purpose: 'FeatureInsight' };
-	readonly requestBodySize: { classification: 'SystemMetaData'; purpose: 'FeatureInsight' };
-	readonly responseBodySize?: { classification: 'SystemMetaData'; purpose: 'FeatureInsight' };
-	readonly statusCode?: { classification: 'SystemMetaData'; purpose: 'FeatureInsight' };
-	readonly errorCode?: { classification: 'SystemMetaData'; purpose: 'FeatureInsight' };
-	readonly count?: { classification: 'SystemMetaData'; purpose: 'FeatureInsight' };
-	readonly source?: { classification: 'SystemMetaData'; purpose: 'FeatureInsight' };
-	readonly searchTextLength?: { classification: 'SystemMetaData'; purpose: 'FeatureInsight' };
+	comment: 'Information about Marketplace query and its response';
```
Comment (sandy081) [inline src/vs/platform/extensionManagement/common/extensionGalleryService.ts:239]: test

**Review by sandy081 — COMMENTED** [review]
@lramos15 Ping

File: src/vs/platform/extensionManagement/common/extensionGalleryService.ts (hunk 1 above)
Comment (sandy081) [inline src/vs/platform/extensionManagement/common/extensionGalleryService.ts:239]: test

**Review by lramos15 — APPROVED** [review]
Thanks so much! Sorry first ping got lost in my inbox

**Pull request merged**